index.html

```diff
@@ -299,17 +213,21 @@ <h1 class="text3">Projects</h1>
         </section>
         <section class="Contact-form" id="contact">
             <img class="img11" src="images/left_topside_icon.svg" alt="Image geometry on the left top side">
-            <form name="Contactform" action="https://formspree.io/f/xbjqawyz" method="POST" aria-required>
+            <form class="mob_form" name="Contactform" action="https://formspree.io/f/xbjqawyz" method="POST" aria-required>
                 <h1 class="text10">Get started</h1>
                 <p class="contact-form-para">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna.</p>
                 <ul class="contact-form-ullist">
                     <li class="contact-form-name">
                         <input type="text" maxlength="30" class="nooutline" id="name" name="user_name" placeholder="Your name" required>
                         <!--div class="sample_text">Anastasiia</div!-->
                     </li>
-                    <li class="contact-form-email"><input type="email" id="mail" class="nooutline" name="user_email" placeholder="Your email" required></li>
+                    <li class="contact-form-email">
+                        <label for="mail"></label>
+                        <input type="email" id="mail" class="nooutline" name="user_email" placeholder="Your email" required onkeydown="validation()">
+                        <small class="chklabel_email" id="error"></small>
+                    </li>
                     <li class="contact-form-message"><textarea maxlength="500" id="msg" class="nooutline" name="user_message" placeholder="Enter your message here..." required></textarea></li>
-                    <li><button class="contact-form-submit" type="submit">Start collaboration</button></li>
+                    <li><button class="contact-form-submit" id="submitnow" type="submit">Start collaboration</button></li>
```

Comment:
- [ ] OPTIONAL
Your form is supposed to display an error message to a user when the email is in capital, and the `submit` button is clicked. Kindly ensure to improve upon this aspect.